runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1880,6 +1882,12 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 		return JNI_EINVAL;
 	}
 
+	if (extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified && extensions->gcThreadCount > 0) {
+		if (extensions->checkpointGCthreadCount > extensions->gcThreadCount) {
```

Comment:
I think there is better condition than `extensions->gcThreadCount > 0`. You can use `extensions->gcThreadCountSpecified` instead.